apps/microtvm/ethosu/run_demo.sh

```diff
@@ -123,32 +123,32 @@ script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 mkdir -p build
 cd build
 
-# Get mobilenet_v1 tflite model
-mobilenet_url='https://storage.googleapis.com/download.tensorflow.org/models/mobilenet_v1_2018_08_02/mobilenet_v1_1.0_224_quant.tgz'
-curl --retry 64 -sSL ${mobilenet_url} | gunzip | tar -xvf - ./mobilenet_v1_1.0_224_quant.tflite
+# Get mobilenet_v2 tflite model
+mobilenet_url='https://github.com/ARM-software/ML-zoo/raw/master/models/image_classification/mobilenet_v2_1.0_224/tflite_int8/mobilenet_v2_1.0_224_INT8.tflite'
```

Comment:
We recently had issues with referencing `master` (https://github.com/apache/tvm/pull/9887) and the model changing, can you pick a SHA1 that guarantees we'll always get the same model?